pkg/systemd/services/services.jsx

```diff
@@ -693,7 +686,14 @@ class ServicesPageBody extends React.Component {
                 const timer_unit = systemd_client[this.props.owner].proxy('org.freedesktop.systemd1.Timer', unitNew.path);
                 timer_unit.wait(() => {
                     if (timer_unit.valid)
-                        this.addTimerProperties(timer_unit, path, unitNew);
+                        if (this.addTimerProperties(timer_unit, unitNew)) {
+                            this.setState(prevState => ({
```

Comment:
This isn't useful by itself, but I'll need it for my big commits to rework the whole state management of the Services page, and I'd like to make sure (run CI) to not break anything.